modules/api/api.py

```diff
@@ -18,15 +19,21 @@ class TextToImageResponse(BaseModel):
     parameters: Json
     info: Json
 
+class ImageToImageResponse(BaseModel):
+    images: list[str] = Field(default=None, title="Image", description="The generated image in base64 format.")
+    parameters: Json
+    info: Json
+
```

Comment:
the exact same as `TextToImageResponse`, but I figured we might want to extend it in the future